src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs

```diff
@@ -132,6 +148,11 @@ internal sealed class MemberExpansion : Expansion
                 }
             }
 
+            if (dynamicViewExpansion != null)
+            {
+                expansions.Add(dynamicViewExpansion);
+            }
```

Comment:
Consider adding a test where there are multiple expansions. (It looks like we're not testing that case for "Results View" either, unfortunately.)